README.md

```diff
@@ -160,13 +160,10 @@ directory.
 
 #### If you have Podman
 
-If you use [Podman][], you can start the test database by following 
+If you have [Podman][podman] installed, you can start the test database by following 
```

Comment:
@DavideD I know it's picky, but this line is also really long compared to the rest of the README

Reply:
Should be fixed now